### ISSUE TITLE
No metadata v14 support

### ISSUE THREAD

**Issue by ptqa** [itering/scale.rb#27]
It's currently impossible to connect to kusama/polkadot networks since there is no support for latest metadata.

**Comment by wuminzhe** [comment]
@ptqa Yes, thank you. I will add the metadata v14 support.

**Comment by s3krit** [comment]
Hey guys, is there likely to be any update on this or has this library been deprecated/abandoned? I am wanting to begin work on a new matrix bot for the Kusama society utilising scale.rb but if you're no longer supporting it I will use something else.
Cheers :)

**Comment by ptqa** [comment]
Yeah, they have abandoned it. They farmed grants and moved over to the next grant.

**Comment by wuminzhe** [comment]
Of course not. Because Itering's projects uses [scale.go](https://github.com/itering/scale.go), scale.rb is not actually used. In addition, scale.rb has many problems in design and development because of being unfamiliar with scale codec and substrate before, so it did not continue. I'm already working on a new implementation: https://github.com/wuminzhe/scale_rb
